examples/estimateBackground.py

```diff
@@ -38,7 +38,7 @@
 
 def getImage():
     imagePath = os.path.join(lsst.utils.getPackageDir("afwdata"),
-                             "DC3a-Sim", "sci", "v5-e0", "v5-e0-c011-a00.sci_img.fits")
+                             "DC3a-Sim", "sci", "v5-e0", "v5-e0-c011-a00.sci.fits")
```

Comment:
Why is this change needed? It doesn't seem to be related either to casting or to Swig container wrappers...